usersDb.js

```diff
@@ -0,0 +1,69 @@
+const pgp = require('pg-promise')()
+const connectionString = process.env.DATABASE_URL || 'postgres://localhost:5432/contacts'
+const db = pgp(connectionString)
+
+const users = {}
+
+users.addNewUser = (email, password) => {
+  return db.query(`
+    INSERT INTO 
+      users (email, password)
+    VALUES
+      ($1, $2)
+  `,
+  [
+    email,
+    password
+  ])
+  .catch(error => error)
+}
+
+users.findUser = (email) => {
+  return db.query(`
+    SELECT
+      *
+    FROM
+      users
+    WHERE
+      email = $1
+  `,
+  [
+    email
+  ])
+  .then(data => data)
+  .catch(error => error)
+}
+
+users.findById = (id) => {
+  return db.query(`
+    SELECT
+      *
+    FROM
+      users
+    WHERE
+      id = $1
+  `,
+  [
+    id
+  ])
+  .then(user => user)
```

Comment:
Same comment as the above, you should return `users[0]` here, since the `db.query` will return an array. The expectation of the function `findById` is to return the user object directly (and not an array)